context/app/static/js/pages/search/Search.jsx

```diff
@@ -86,7 +87,10 @@ function Search({ title }) {
   return (
     <>
       <SearchHeader component="h1" variant="h2">
-        {title}
+        <SearchEntityHeader data-testid="entity-header">
+          <StyledSvgIcon component={entityIconMap[`${title.slice(0, title.length - 1)}`]} color="primary" />
```

Comment:
I think I would prefer not to rely on the title here. There is `type` defined above which gets the `entity_type[0]` param from the url search params. You could add `capitalizedType` and adjust `type` and use `capitalizedType` here instead of relying on `title`.

```
  const capitalizedType = searchParams.get(typeParam) || '';
  const type = capitalizedType.toLowerCase();
```

Reply:
I see, okay will do.